src/drivers/boards/auav-x21/board_config.h

```diff
@@ -210,6 +210,9 @@
 
 #define BOARD_NAME "AUAV_X21"
 
+// Run the FMU task on its own thread to reduce jitter
+#define PX4FMU_TASK
```

Comment:
@kd0aij - 

With non zero/non blank values there is less room for ambiguity and any error of the type in shown in #1619     

I am not sure there is a guide you can follow. As a rule of thumb I try not to deviate from what I see in a file for format, content and try to use what patterns I see there first before inventing new ones. 

For instance: comments in the file are ```/* */``` and all the #defines have a value
 (the new lines do not)

I hope this helps you.